<launch>

    <!-- other args -->
  <arg name="standalone" default="true" />
  <arg name="debug" default="false" />
  <arg name="variant" default="" />

  <arg     if="$(eval arg('standalone') or arg('debug'))" name="nodelet" value="standalone" />
  <arg unless="$(eval arg('standalone') or arg('debug'))" name="nodelet" value="load" />
  <arg     if="$(eval arg('standalone') or arg('debug'))" name="nodelet_manager" value="" />
  <arg unless="$(eval arg('standalone') or arg('debug'))" name="nodelet_manager" value="nodelet_manager" />

  <arg     if="$(arg debug)" name="launch_prefix" value="debug_roslaunch" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />

  <arg name="file" default="" />

  <node name="simulation_test" pkg="nodelet" type="nodelet" args="$(arg nodelet) task_02_evaluation/SimulationTest $(arg nodelet_manager)" output="screen" launch-prefix="$(arg launch_prefix)">

    <rosparam file="$(find task_02_evaluation)/config/simulation_test.yaml" />

    <param name="file" value="$(arg file) "/>

    <param name="variant" value="$(arg variant) "/>

      <!-- subscribers -->
    <remap from="~diagnostics_in" to="/task_02_wrapper/diagnostics" />

  </node>

</launch>
